app/components/search/Sidebar/Sidebar.module.scss

```diff
@@ -38,6 +38,7 @@
     align-self: flex-start;
     border-right: 0;
     background-color: #fff;
+    z-index: $z-index-filters-menu;
```

Comment:
**note:** Discovered a bug where the filters were not on top.

Reply:
Speaking of z-indexes. The category dropdown appears to have a higher z-index than the navigation bar. If that would be a quick fix, might make sense to put it in this PR

![image](https://github.com/user-attachments/assets/c35407c9-e2dd-4ff5-88ff-aec269971178)

Reply:
What browser/resolution is that? I don't even know how to get in that state 😅